### ISSUE TITLE
KubeCon EU Maintainer Summit 2025 - Team Staffing

### ISSUE THREAD

**Issue by mfahlandt** [kubernetes/community#8217]
The 2025 NA Maintainer Summit in London will need staffing for its roles and shadows. 

It will happen at 31st of March 2025 in London Excel London
Royal Victoria Dock, 1 Western Gateway, London E16 1XL, United Kingdom

Event Lead will be @mfahlandt 

We will also need shadows that could potentially take on leading those roles in the future. Roles and shadows will be listed below when they are filled out. With limited exception, shadows must have Kubernetes org membership. Shadows will be accepted at the discretion of the lead.

To clarify roles and responsibilities, we are adding a role of "Volunteer" under each area. The Shadow bears expectations to lead one of the events-team roles in an upcoming summit and to handle all lead related activities in the absence of the Lead. It may be helpful to think of the Shadow role as an understudy position. Volunteers are helpers that may be assigned tasks from the Lead. Shadows and Volunteers, with limited exception, must be members of the Kubernetes GitHub organization. Leads are still responsible for accepting shadows and volunteers for their area. Volunteers, please communicate your desire to be a Shadow to the Lead of your area. Leads will inform me of their Shadows and I will update the issue.

As this will be a first time Maintainer Summit we still will need to figure out certain roles and what to do about it. Comments and critique is welcome. 
For now, we decided to go with a smaller stack of roles. 
The need for a meeting is still to be discussed. 

@natalisucks & @angellk & @mfahlandt are the overall chairs for the maintainer summit. 

Thank you for your willingness to help out. We could not hold these events without you.

The program committee for the Kubernetes track was pre chosen by ContribEx

Committee Chairs: @katcosgrove, @jberkus
Committee Members: @ArvindParekh, @jeremyrickard, @tabbysable, @qedrakmar, @sandeepkanabar, @savitharaghunathan, @soltysh, @stmcginnis 

[Comms](https://github.com/kubernetes/community/blob/master/events/events-team/marketing/README.md)
Lead: @fykaa @ArvindParekh 
Shadow: @wendy-ha18, @lavishpal 
Volunteers:
Website:
Design: 

[SIG Meet & Greet (Preferred onsite in London)](https://github.com/kubernetes/community/blob/master/events/events-team/content/meet-and-greet.md)
Lead: @reylejano 
Shadow: @sreeram-venkitesh, @inductor
Volunteers:
Emeritus Advisor:

/assign @mfahlandt 
/sig contributor-experience
/area maintainer-summit
/area EU-summit

**Comment by mfahlandt** [comment]
We might need a new label

**Comment by reylejano** [comment]
I can lead the SIG Meet & Greet, I shadowed SIG Meet & Greet for KubeCon NA 2024
I'm also open for another person to lead SIG Meet & Greet if they prefer

**Comment by sreeram-venkitesh** [comment]
I'd like to shadow for SIG Meet and Greet :) I'll be present at London.

**Comment by tokt** [comment]
@lauralorenz was shadowing awards at NA24 and I'd like to nominate Laura as a lead for Awards in EU25.
(if we're going to do awards, that's still under discussion). 

Happy to shadow awards/registration and/or anywhere else.

**Comment by lavishpal** [comment]
I'd like to shadow for SIG Meet and Greet :)
However, I won’t be able to attend KubeCon in person this time. I’m still very eager to contribute and learn through the process remotely, if that’s possible

**Comment by satrox28** [comment]
I am very much Interested to volunteer on the KubeCon EU. I will be
attending the event and a Kubestronaut.

**Comment by qedrakmar** [comment]
Will there be a need for Operations folks this time around?

**Comment by natalisucks** [comment]
Please note everyone: to be part of the Planning Committee for the Maintainer Summit for Kubernetes, you **must** be an organization member. @mfahlandt can you please make sure to emphasize this in the issue description?

@satrox28 unfortunately, you will need to be a Kubernetes organization member to help with planning for the Summit, but the good news is that there is still time to become a member via regular contributions. See here for more information: https://github.com/kubernetes/community/blob/master/community-membership.md#member

Whoever puts their hand up as Comms Lead/shadows, please note that you will be working with myself and @angellk directly so that we can streamline comms across all projects and CNCF channels. Also, while we will still utilize the #contributor-summit Slack channel on k8s Slack, there is the intention to create a #maintainer-summit channel on the CNCF Slack for streamlined comms across all projects, which will also need to be monitored on the day by the team for questions.

**Comment by mfahlandt** [comment]
We had in the past folks who helped with the summit without ORG membership - they just could not join. For Meet & Greet, I agree this does not necessarily make sense. However, for Comms for drafting comms, this works. 

@qedrakmar currently there is no plan / requirement for Day of Ops folks :)

**Comment by rayandas** [comment]
I would be present in London so would like to shadow for SIG Meet & Greet.

**Comment by jberkus** [comment]
@mfahlandt  can you explain what we're planning for Awards in London?   In the past, we haven't done awards for Kubecon EU.

**Comment by mfahlandt** [comment]
As discussed in the retro of the kcs na summit there was the proposal to hand out awards in Europe or offer sigs to decide where to give awards. 
As EU is so close @tokt volunteered to ping the sigs who have not awarded any in na to do so in EU as an option. Afterwards there was planned to have a discussion with SIG leads on this.

See the action items of the retro doc for this.

One of the reasons was - EU kcs was surpassing NA in attendees for the previous years

**Comment by stmcginnis** [comment]
> One of the reasons was - EU kcs was surpassing NA in attendees for the previous years

I also think with the increase of KubeCon events, we may need to start thinking about regional awards or some way to be able to reach the folks that can only attend limited local(ish) events.

**Comment by RA489** [comment]
I have been shadow in KubeCon NA 2024 for SIG meet and greet. Happy to help as Lead/shadow/Volunteer

**Comment by jberkus** [comment]
> One of the reasons was - EU kcs was surpassing NA in attendees for the previous years

Yeah, I'm the one who brought this up.  Opinions on it seemed kinda negative, so I didn't think we were doing it.

**Comment by AryanParashar24** [comment]
I'd love to be the Lead/shadow/volunteer for SIG Meet & greet.

**Comment by jberkus** [comment]
@mfahlandt @tokt I'm happy to help with Awards then.  In whatever role.

**Comment by tokt** [comment]
So I've reached out to a few SIGs (there were 6 SIGs without awardees at KCSNA24) and there hasn't been a lot of enthusiasm about awards in London. (Not all SIGs have responded yet, but those who did - prefer to keep things at NA events).
Our options are:
- keep things as is and handle awards at KubeCon NA25
- ask more SIGs if they'd perhaps prefer a ceremony at EU-25 instead of NA-25?
- ??

**Comment by stmcginnis** [comment]
I would vote for keeping things simple then, and only doing community awards with the NA event like we've done so far.

**Comment by mikejoh** [comment]
Hi! I was a volunteer at the Contributor Summit EU last year, it would be a pleasure to help out again this year!

**Comment by AryanParashar24** [comment]
I have applied to shadow for the docs and communication role and now i would love to contribute more for the KubeCon EU SIG Meet & Greet

**Comment by ArvindParekh** [comment]
I previously shadowed the comms role last cycle under @kaslin, and I would love to co-lead comms (as per our discussion on #summit-staff slack) along with @fykaa! :)
However, I'm not sure if I'll be present on-site during the summit.

**Comment by inductor** [comment]
I can do an onsite volunteer wherever is lacking of people 👍

**Comment by mfahlandt** [comment]
Hey Folks, 

a big apologies for not being as responsive. There is no excuse.

Thanks, @fykaa & @ArvindParekh - confirming as leads for comms

Thank you, @reylejano, for meet and greet

Please confirm additional shadows. 

@AryanParashar24 unfourtunatly you need to be Kubernetes ORG member to support here. Although @reylejano can do an exception as Meet & Greet will happen at KubeCon itself

**Comment by lavishpal** [comment]
Hey @mfahlandt, I'd like to volunteer for comms :)

**Comment by mfahlandt** [comment]
@lavishpal @fykaa and @ArvindParekh need to decide this :)

**Comment by lavishpal** [comment]
Thank you for the update! I’ll wait to hear back from  @fykaa  and @ArvindParekh . 
Please let me know if there’s anything else I need to do in the meantime!

**Comment by reylejano** [comment]
@inductor , we would love your assistance on-site for the SIG Meet & Greet, let me us know if you're interested.
On-site help involves setting up the table signs for the various SIGs and Working Groups before the Meet and Greet, helping guide attendees to the SIG or WG they're interested in

**Comment by iheartNathan** [comment]
@reylejano I'm also interested in the onsite assistance for the SIG Meet & Greet.

**Comment by vellankikoti** [comment]
Hello,  I would like to volunteer and shadow for communications team!

**Comment by kendallnelson** [comment]
I realize I am very late to the party, but I recently got confirmation that I will be present at the event and as such I am happy to help out onsite/ with operations!
